client/data/deposits/test/resolvers.js

```diff
@@ -57,36 +59,54 @@ const filterQuery = {
 };
 
 describe( 'getDeposit resolver', () => {
-	let generator = null;
+	describe( 'on', () => {
```

Comment:
This diff and below looks a lot but I just wrap the existing unit test in another layer of `describe()`.